st2api/st2api/controllers/v1/pack_configs.py

```diff
@@ -63,3 +77,28 @@ def get_one(self, pack_ref):
         """
         # TODO: Make sure secret values are masked
         return self._get_one_by_pack_ref(pack_ref=pack_ref)
+
+    @request_user_has_permission(permission_type=PermissionType.PACK_CREATE)
+    @jsexpose(body_cls=ConfigUpdateRequestAPI, arg_types=[str])
+    def put(self, pack_uninstall_request, pack_ref):
+        """
+            Create a new config for the action.
+
+            Handles requests:
+                POST /configs/<pack_ref>
+        """
+
+        try:
+            config_api = ConfigAPI(pack=pack_ref, values=vars(pack_uninstall_request))
+            config_api.validate(validate_against_schema=True)
+        except jsonschema.ValidationError as e:
+            raise ValueValidationException(str(e))
+
+        config_content = yaml.safe_dump(config_api.values, default_flow_style=False)
+
+        configs_path = os.path.join(cfg.CONF.system.base_path, 'configs/')
```

Comment:
Minor thing - I believe we do this in multiple places so we should probably and and use a utility function to retrieve a path. Similar as we do for other paths.